kategory/src/main/kotlin/kategory/data/Const.kt

```diff
@@ -0,0 +1,39 @@
+package kategory
+
+fun <A, T> ConstKind<A, T>.value(): A = this.ev().value
```

Comment:
If it's doable we can have this as an optional arg for the best of both worlds in user defined datatypes.
```kotlin
@higherkind(delegation = DelegationStrategy.ALL)
```